src/createXcmTypes/util/getAssetId.spec.ts

```diff
@@ -14,30 +14,30 @@ describe('getAssetId', () => {
 		it('Should correctly return the integer assetId when given a valid native system chain token symbol', async () => {
 			const expected = '10';
 
-			const result = await getAssetId(systemAssetsApi._api, registry, 'USDC', 'statemine', 2, false);
+			const result = await getAssetId(systemAssetsApi.api, registry, 'USDC', 'statemine', 2, false);
 
 			expect(result).toEqual(expected);
 		});
 
 		it('Should correctly return the integer assetId when given a valid native system chain token assetId', async () => {
 			const expected = '8';
 
-			const result = await getAssetId(systemAssetsApi._api, registry, 'RMRK', 'statemine', 2, false);
+			const result = await getAssetId(systemAssetsApi.api, registry, 'RMRK', 'statemine', 2, false);
 
 			expect(result).toEqual(expected);
 		});
 
 		it('Should error when an asset id symbol is given that is not present in the registry or chain state', async () => {
 			await expect(async () => {
-				await getAssetId(systemAssetsApi._api, registry, 'hello', 'statemine', 2, false);
+				await getAssetId(systemAssetsApi.api, registry, 'hello', 'statemine', 2, false);
 			}).rejects.toThrowError('assetId hello is not a valid symbol or integer asset id');
```

Comment:
```suggestion
			}).rejects.toThrow('assetId hello is not a valid symbol or integer asset id');
```
I noticed that `toThrowError` appears crossed out in this file and in other files of the code base. If I click on it, then it brings me to a `jest` index file where it mentions `@deprecated in favor of `toThrow``.